apps/validation.py

```diff
@@ -155,27 +170,45 @@ def get_flags(paper):
                 reset=page[area].clear,
             )
             async for result in regen:
-                if not has_paper_validation(result):
-                    div = html(result)
-                    divFlags = get_flags(result)
-                    valDiv = H.div["validationDiv"](
-                        div,
-                        H.button["button"](
-                            "Validate",
-                            onclick=(
-                                lambda event, paper=result: validate_button(
-                                    paper, 1
-                                )
+                if seeFlagged:
+                    if has_paper_validation(result):
+                        div = html(result)
+                        divFlags = get_flags(result)
+                        valDiv = H.div["validationDiv"](
+                            div,
+                            H.button["button"](
+                                "Undo",
+                                onclick=(
+                                    lambda event, paper=result: unValidate(
+                                        paper
+                                    )
+                                ),
+                            ),
```

Comment:
Usually, if the person doing the validation made a mistake, they will want not only to undo, but also to correct their mistake by clicking the other button. So we should have three buttons here: undo, validate, invalidate.

Reply:
Done in [2ab8dd4](https://github.com/mila-iqia/paperoni/pull/21/commits/2ab8dd46f8692a247fe644605e005750cbf89620)